### PR TITLE
View archived petitions

### PR DESCRIPTION
**Pull request by alanth**

### CODE REVIEW

**Comment by alanth** [comment]
Shorten the strings a little with `\` on to new lines 
- You could make that archive message into a let so it can be reused.
- File name ideally for consistency should be lowercased
- Maybe move the last arg to Factory.create to next line otherwise its a lot of scrolling but not critical
- Also no need to use @petition when it is local to the test only - change that to not have the `@` sign
- Make the test a little more descriptive I am not sure what it is testing at first glance of reading the description...

File: spec/features/view_archived_petition_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'View archived petition', type: :feature do
+  let(:archive_message) {'This petition has been archived It was submitted during the 2010–2015 Conservative – Liberal Democrat coalition government'}
+  let(:petition) {@petition = FactoryGirl.create(:archived_petition, :closed, title: 'Spend more money on Defence')}
```
Comment (jnormington) [inline spec/features/view_archived_petition_spec.rb]: Remove the `@petition =` assignment its not required.

File: spec/features/view_archived_petition_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    expect(current_path).to eq(archived_petition_path(petition))
+    expect(page).to have_content(petition.title)
+    expect(page).to have_content(petition.description)
+    expect(page).to have_title('Spend more money on Defence'.to_s)
+    expect(page).to have_css("li.meta-deadline", :text => "Deadline " + petition.closed_at.strftime("%e %B %Y").squish)
+    expect(page).to have_content(archive_message)
+  end
+  
+  it 'that contains urls, email addresses and html tags' do
+    @petition = FactoryGirl.create(:archived_petition, :closed, title: 'Spend more money on Defence', description: "<i>We<i> like http://www.google.com and bambi@gmail.com")
```
Comment (jnormington) [inline spec/features/view_archived_petition_spec.rb]: This assignment is never used and probably could be removed

Comment (alanth) [inline spec/features/view_archived_petition_spec.rb]: It's required to add the description which is used later on in the test, links etc.

Comment (jnormington) [inline spec/features/view_archived_petition_spec.rb]: In that test `@petition` is never used not that I can see...

File: spec/features/view_archived_petition_spec.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+RSpec.describe 'View archived petition', type: :feature do
+  let(:archive_message) {'This petition has been archived It was submitted during the 2010–2015 Conservative – Liberal Democrat coalition government'}
+  let(:petition) {FactoryGirl.create(:archived_petition, :closed, title: 'Spend more money on Defence')}
+  let(:rejected_petition) {FactoryGirl.create(:archived_petition, :rejected, title: "Please bring back Eldorado", reason_for_rejection: "<i>We<i> like http://www.google.com and bambi@gmail.com")}
+  
+  it 'that is closed' do
+    visit archived_petition_url(petition)
+    expect(page).to have_content(petition.title)
+    expect(page).to have_content(petition.description)
+    expect(page).to have_title('Spend more money on Defence'.to_s)
```
Comment (jnormington) [inline spec/features/view_archived_petition_spec.rb]: No-need to call `#to_s` on an already string. Its over a few places

**Pull request merged**